packages/ui/src/components/TimelineItem/TimelineItem.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Center, FlexProps, useBreakpointValue } from '@chakra-ui/react';
+
+import { Typography } from '../Typography';
+
+export interface TimelineItemProps extends FlexProps {
+  step: number;
+  date: string;
+  information: string;
+}
+
+export const TimelineItem: React.FC<TimelineItemProps> = ({ step, date, information, ...props }) => {
+  const boxPadding = useBreakpointValue({ base: '18px', md: '32px' } as const) ?? '18px';
```

Comment:
💭 You can define the default breakpoint as the second argument of the `useBreakpointValue`.

Reply:
I did it, but still returned 'const boxPadding: "18px" | "32px" | undefined'